<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Info</class>
 <widget class="QWidget" name="Info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Info about a RAID</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>631</width>
     <height>223</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="raid_details">
    <property name="sizeConstraint">
     <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
    </property>
    <property name="horizontalSpacing">
     <number>6</number>
    </property>
    <property name="verticalSpacing">
     <number>20</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="select_raid_label">
      <property name="text">
       <string>Select RAID:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="select_raid"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="raid_path_label">
      <property name="text">
       <string>RAID Path:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="raid_path">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="raid_level_label">
      <property name="text">
       <string>RAID Level:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="raid_level">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="select_size_label">
      <property name="text">
       <string>RAID Size:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="raid_size">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="raid_state_label">
      <property name="text">
       <string>RAID State:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="raid_state">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="raid_devices_label">
      <property name="text">
       <string>RAID Devices:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="raid_devices">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="OK_button">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>440</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>false</bold>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::ContextMenuPolicy::ActionsContextMenu</enum>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QWidget" name="row1" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>631</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="active_label">
      <property name="text">
       <string>Active devices:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="active_devices">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="working_label">
      <property name="text">
       <string>Working devices:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="working_devices">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="row2" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>631</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="failed_label">
      <property name="text">
       <string>Failed devices:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="failed_devices">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="spare_label">
      <property name="text">
       <string>Spare devices:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="spare_devices">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
